src/atoms/connectWalletButton/connectWalletButton.styles.ts

```diff
@@ -0,0 +1,36 @@
+import { Styled } from '../../theme';
+import { ConnectButtonProps } from './types';
+
+export const ConnectButton = Styled.button<ConnectButtonProps>`
+  svg {
+    cursor: pointer;
+    fill: ${(props) => props.theme.textShades.SHADE_MINUS_3}!important;
+  };
+  font-size: 14px;
+  line-height: 24px;
+  padding: 12px;
+  display: flex;
+  gap: 4px;
+  align-items: center;
+  font-weight: 700;
+  color: ${(props) => props.theme.textShades.SHADE_MINUS_3};
+  border-radius: 12px;
+  background-color: transparent;
+  border: ${(props) => `1px solid ${props.theme.colors.primary.UWL_BLUE}`};
+  cursor: ${(props) => (props.disabled ? 'default' : 'pointer')};
+  .button__group {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 5px;
+  };
+
+  &:hover {
+  color: ${(props) => props.theme.colors.system.WHITE};
+  background: ${(props) => (props.account !== null ? props.theme.colors.system.RED : props.theme.colors.primary.UWL_BLUE)};
```

Comment:
I noticed that in the connect button props the account is optional and sometimes rather than checking `account !== null ? ...` you just do `account ? ...` i think its better to keep it consistent and rather than checking `account !== null ? ...` always do `!account ?...` or `account ?... ` it covers more cases and since in props account is optional someone can pass `undefined` rather than `null` which would cause a bug in this case.

Reply:
Agreed